docs/playbooks/how-to-join-a-dao-community.md

```diff
@@ -5,7 +5,7 @@ title: 📚 How to Join DAO Community
 by @Techniciann9ne
 
 
-- How non developers can make an impact in the dao space
+- How non-developers can make an impact in the DAO space
```

Comment:
It would be good to see this as a level 3 heading using `### How non-developers can make an impact in the DAO space` rather than a `-` which makes a list item.